### PR TITLE
CSV Idiomatic Errors

### PR DESCRIPTION
**Pull request by cpheps**
- Changed errors in CSV parser operator to have lowercase first words to be idiomatic.
- Removed some `fmt.Errorf` in favor of `errors.New` where formatting wasn't needed to create error

### CODE REVIEW

**Comment by jsirianni** [comment]
@cpheps Let me know when CLA is handled and I will trigger CI 👍

**Comment by cpheps** [comment]
> 

@jsirianni will do. @djaglowski is working on getting me signed up I'll follow up with you when that's done.

**Review by djaglowski — COMMENTED** [review]
LGTM, but waiting on EasyCLA approval

**Review by djaglowski — APPROVED** [review]

**Pull request merged**
